src/adapter/src/coord/dataflows.rs

```diff
@@ -637,7 +637,7 @@ fn eval_unmaterializable_func(
         }
         UnmaterializableFunc::CurrentTimestamp => pack(Datum::from(session.pcx().wall_time)),
         UnmaterializableFunc::CurrentUser => pack(Datum::from(session.user())),
-        UnmaterializableFunc::MzClusterId => pack(Datum::from(state.config().cluster_id)),
+        UnmaterializableFunc::MzEnvironmentId => pack(Datum::from(state.config().environment_id)),
```

Comment:
This is getting a bit out of my depth, but -- given that this is going to be constant for any mz deployment over its entire lifetime, it would be safe to use in a materialized view, yeah?

Reply:
Yeah, good point, it would be! I guess that was true before, too, with `mz_cluster_id`. I don't mind it terribly though, tbh, since it's erring on the side of too conservative. Probably more hassle than it's worth to fix at th emoemnt.

Reply:
Yeah, makes sense! Definitely agree there's no rush; just thinking through it.